src/idsk/components/footer-extended/template.njk

```diff
@@ -1,131 +1,297 @@
 {% from "details/macro.njk" import govukDetails %}
-<div class="idsk-footer__meta idsk-footer-extended {{ params.classes if params.classes }}">
-    <div class="govuk-width-container">
-        {% if params.firstTitle %}
-            <div class="govuk-grid-row">
-                <div class="govuk-grid-column-two-thirds two-thirds">
-                    <h3 class="govuk-heading-m ">
-                        {{ params.firstTitle }}
-                    </h3>
-                    <div class="govuk-grid-row small_subtitle">
-                        <div class="govuk-grid-column-one-half bottom-space">
-                            {% for i in range(0, params.numberOfSubtitles) -%}
-                                <li class="idsk-footer__inline-list-item">
-                                    <a class="idsk-footer__link" href="{{firstSubtitle.href}}">{{ params.firstSubtitle }}</a>
-                                </li>
-                            {%- endfor %}
+<div data-module="idsk-footer-extended">
+    <div id="feedback-question" class="heading-feedback feedback-text ">
+        <div class="govuk-width-container feedback-container">
+            <div id="idsk-info-question" class="govuk-grid-row">
+                <div class="footer-usefull-question">
+                    <a class="feedback-question-info-usefull">
+                        Boli tieto informácie pre vás užitočné?
```

Comment:
ak to nevyvolava ziadnu akciu, urcite by som tu nedal <a>.. ale span..to iste inde